packages/block-directory/src/components/downloadable-block-info/style.scss

```diff
@@ -0,0 +1,24 @@
+.block-directory-downloadable-block-info__content {
+	font-size: 14px;
```

Comment:
14px is a nice font size, but I would still suggest using $default-font-size here so that the font size is not only consistent with the rest of the editor, but indeed will be properly affected if we choose to bump the default 13px font size in the variables sheet at some point in the future.